pkg/webhook/clusterset/validatingWebhook.go

```diff
@@ -110,7 +151,7 @@ func (a *AdmissionHandler) validateCreateRequest(request *v1.AdmissionRequest) *
 	labels := obj.GetLabels()
 	clusterSetName := ""
 	if len(labels) > 0 {
-		clusterSetName = labels[clusterSetLabel]
+		clusterSetName = labels[clustersetutils.ClusterSetLabel]
```

Comment:
nit, ClusterSetLabel is in the api repo.

Reply:
done